templating/namespaced_paths.rst

```diff
@@ -14,7 +14,7 @@ Take the following paths as an example:
 
 .. code-block:: twig
 
-    {% extends "AppBundle::layout.html.twig" %}
+    {% extends "AppBundle:layout.html.twig" %}
```

Comment:
I'm probably missing something here. I've just tested this in a fresh Symfony Demo app:

```twig
{# this WORKS and renders src/AppBundle/Resources/views/layout.html.twig #}
{% extends "AppBundle::layout.html.twig" %}

{# this DOES NOT WORK and just displays a "unable to find template" error #}
{% extends "AppBundle:layout.html.twig" %}
```

So I still think that the proposed change is wrong.